src/sagas/search.js

```diff
@@ -0,0 +1,38 @@
+import { delay } from 'redux-saga'
+import { takeLatest, all, put, call } from "redux-saga/effects";
+import search from '../actions/search'
+import axios from "axios";
+import api from '../api';
+
+const { types: { SEARCH_GET }, creators: { fetchedSearch } } = search
+const { HOST, endpoints: { SEARCH }, KEY } = api
+
+function* watchSearch() {
+    yield all([
+        takeLatest(SEARCH_GET, getSearch)
+    ])
+}
+
+function* getSearch(action) {
+    try {
+        yield call(delay, 500)
```

Comment:
Why are you using `delay`?

Reply:
I tried to use takeLastest without delay, but takeLastest only cancel if there are a task in process, when I tried to use it, the task of the input ends Before the next event comes; for that reason I use delay to cancel the task correctly,  But I will try again